meta-ostro/conf/distro/include/stateless.inc

```diff
@@ -498,7 +510,7 @@ passwd:         files altfiles
 group:          files altfiles
 shadow:         files altfiles
 
-hosts:          files altfiles dns
+hosts:          files mymachines resolve myhostname altfiles dns
```

Comment:
What is the reasoning for this order? Please explain in the commit message, it currently does not describe this change at all.

FWIW, "files" and "altfiles" are closely related (/etc/hosts and /usr/.../hosts), so if "files" comes before "mymachines resolve myhostname", then "altfiles" might also have to come first.